### DISCUSSION TITLE
Ask you questions here:

### DISCUSSION THREAD

**Discussion by AammarTufail** [AammarTufail/machinelearning_ka_chilla#2] [Q&A]
# **Ask you questions here as follows:**

1. Take a screenshot of the error with windows snipping tool (no mobile picture is approved)
2. Refine your question in an easy and understandable way
3. Post here and wait for the answer.

**Comment by Pyali786** [comment]
wow sir allah app ko lame uper de

**Reply by Pyali786** [reply]
Thanks
